src/services/errorTracking/adapters/Sentry.ts

```diff
@@ -20,6 +20,7 @@ export const SentryAdapter = (config: Config): TrackerMethods => {
           "ResizeObserver loop completed with undelivered notifications",
           // TODO: rmoeve after Cypress migation
           "ResizeObserver loop limit exceeded",
+          "Cannot read properties of undefined (reading 'holder')",
```

Comment:
question: This is easy fix and a last resort, did you try to handle this case in code, like in `ClientEditorCore` class/

Reply:
Yes, I did try to handle this error in `ClientEditorCore` but it didn't help